### DISCUSSION TITLE
Bug : not working in 1909. Showing 12007 error while launching 20.04 ubuntu

### DISCUSSION THREAD

**Discussion by anirugu** [microsoft/WSL#6519] [General]
I don't have store in my windows. I install using the appx and enable the linux subsystem in program and features. Can someone tell me what is means of this error.
![image](https://user-images.githubusercontent.com/35772421/106700920-cf170d80-660b-11eb-9b88-09f8e20a2d37.png)

I read somewhere about proxy so I turn off automatic proxy in windows 10. I still have the same issue.

**Comment by Zephons** [comment]
Hello. Did you manage to solve the problem? I am having the same issue. Thank you!

**Reply by anirugu** [reply]
I have no idea about  the problem, It's almost more than 3 years now.
